include/bluetooth/mesh/access.h

```diff
@@ -511,6 +511,12 @@ struct bt_mesh_model {
 	/* Model callback structure. */
 	const struct bt_mesh_model_cb * const cb;
 
+#ifdef CONFIG_BT_MESH_MODEL_EXTENSIONS
```

Comment:
I think you need to define this for doxygen in order to get it included in the online documentation. But I don't know where exactly. @jhedberg You know this right?

Reply:
Look for "PREDEFINED" in `doc/zephyr.doxyfile.in`